engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -3902,6 +3919,7 @@ private Option add(Option opt)
     final Option originalAppID = add(OptionBuilder.withArgName("application id").hasArg().withDescription("Specify original application identifier for restart.").create("originalAppId"));
     final Option exactMatch = add(new Option("exactMatch", "Only consider applications with exact app name"));
     final Option queue = add(OptionBuilder.withArgName("queue name").hasArg().withDescription("Specify the queue to launch the application").create("queue"));
+    final Option tags = add(OptionBuilder.withArgName("comma separated tags").hasArg().withDescription("Specify the tags for the application").create("tags"));
```

Comment:
For the apache commons cli library that we are using, the way to specify multiple values is like:
```
-tag a -tag b -tag c
```
But I think it's easier for the user to say `-tags a,b,c`, even though this would implicitly disallow the use of commas in tags. I don't have strong opinions either way though.